assets/css/main.css

```diff
@@ -79,26 +79,30 @@ Content
 */
 
 .role-header,
-.role-header-healer,
-.role-header-tank,
-.role-header-dps {
+.role-header-healers,
+.role-header-tanks,
+.role-header-melee,
+.role-header-ranged,
+.role-header-casters {
```

Comment:
I think this is where I pull out the 🔪 and point out that these changes are separate from generating editor config

Reply:
I can understand your concerns, but please also keep in mind that the admin cp changes finally integrated the roles correctly. If I would not integrate this change into the PR, it would mean that we have visually speaking a more "broken" state that we have without this PR. So the 🔪 might be a bit harsh here. Take a ⚾ or so instead.

Reply:
I think my concerns lie in a few places:

1. I'm not sure we're going to land these editor changes in production before soft launch
2. The editor and the theme are two separate components in terms of purpose. Absolutely – one does influence the other, but the theme can absolutely work without the editor.
3. Combined PRs can hold up changes that we need because we need to wait for all part(s) to be ready for merging.